test/uk/gov/hmrc/tai/connectors/TaxAccountConnectorSpec.scala

```diff
@@ -137,7 +137,7 @@ class TaxAccountConnectorSpec extends PlaySpec with WireMockHelper with MockitoS
           val featureTogglesConfig = mock[FeatureTogglesConfig]
           val url = {
             val path = new URL(iabdUrlConfig.desIabdEmploymentUrl(nino, taxYear, NewEstimatedPay.code))
-            s"${path.getPath}"
+            "${path.getPath}"
```

Comment:
The value of `url` will be "${path.getPath}" and not the actual value of `path.getPath`, this is because you have removed the proceeding `s` which enables interpolation. The return value should be `path.getPath` with no quotes and no `${}`